backend/src/api/vulnerabilities.ts

```diff
@@ -190,22 +190,25 @@ class VulnerabilitySearch {
         .orderBy(sort, this.order);
     }
 
-    if (pageSize !== -1) {
-      qs = qs.offset(pageSize * (this.page - 1)).limit(pageSize);
-    }
-
     await this.filterResultQueryset(qs, event);
     if (!isGlobalViewAdmin(event)) {
       qs.andWhere('organization.id IN (:...orgs)', {
         orgs: getOrgMemberships(event)
       });
     }
 
+    if (groupBy) {
+      const tempResults = await qs.getRawMany();
```

Comment:
My one concern here is that this would make the dashboard super slow, because we would no longer be adding limits to the query here. Would either @mbachtell-nais or @lfilip-ivaal be able to test this with the large amount of sample data to see if it would unreasonably slow the dashboard down?